src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1651,8 +1651,9 @@
     <Field Name="Type" Type="TypeSymbol" Override="true" Null="always"/>
   </Node>
 
-  <!-- Special node, used only during Data Flow Pass -->
+  <!-- Special node, used only during nullability flow analysis -->
   <Node Name="BoundValuePlaceholder" Base="BoundExpression">
```

Comment:
Not related to this PR: Should this node derive from BoundValuePlaceholderBase? Also, I'd try to give it a specific name (see other placeholder nodes above, such as deconstruction, out var, etc).

Reply:
Added `// PROTOTYPE` comment.